GNUmakefile

```diff
@@ -58,20 +58,29 @@ SHELL-NAME := makes go-yaml
 include $(MAKES)/clean.mk
 include $(MAKES)/shell.mk
 
-MAKES-CLEAN := $(dir $(YTS-DIR)) $(GOLANGCI-LINT)
+MAKES-CLEAN += $(GOLANGCI-LINT)
+
+MAKE := $(MAKE) --no-print-directory
 
 v ?=
 count ?= 1
 
 
 # Test rules:
-test: $(GO-DEPS)
+check:
+	$(MAKE) fmt
+	$(MAKE) tidy
+	$(MAKE) lint
+	$(MAKE) vet
+	$(MAKE) test
+
+test: test-unit test-cli test-yts-all
+
+test-unit: $(GO-DEPS)
 	go test$(if $v, -v)
 
 test-data: $(YTS-DIR)
 
-test-all: test test-yts-all
-
 test-yts: $(GO-DEPS) $(YTS-DIR)
 	go test$(if $v, -v) ./yts -count=$(count)
```

Comment:
I would have preferred if this PR was built on top of #158

We validated we would remove make fumpt and make vet 🙄